Updating "Masking out the solar disk" example (#5114)
sunpy/sunpy: changelog/5114.doc.rst

> Updated the `Masking out the solar disk` example by generating the mask using `sunpy.map.coordinate_is_on_solar_disk` instead of calculating normalized radial position for each pixel.

Reviewer: ```suggestion
Updated the :ref:`sphx_glr_generated_gallery_computer_vision_techniques_mask_disk.py` example to generate the mask using :func:`sunpy.map.coordinate_is_on_solar_disk`.
```
